core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -785,13 +741,6 @@ public static function resolveFlaggedUrl(&$url)
 	 */
 	public static function convertEncoding($str, $to, $from=null)
 	{
-		if ($str !== null && !\is_scalar($str) && !(\is_object($str) && method_exists($str, '__toString')))
-		{
-			trigger_deprecation('contao/core-bundle', '4.9', 'Passing a non-stringable argument to StringUtil::convertEncoding() has been deprecated an will no longer work in Contao 5.0.');
-
-			return '';
-		}
-
```

Comment:
Again, we should either add a type hint or throw an exception to enforce the type.